skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -1015,5 +1042,82 @@ def test_from_dict_to_dict_roundtrip(self):
         self.assertIs(d1[42], d2[42])
 
 
+class TestAppend(unittest.TestCase):
```

Comment:
I'm fine with mixing. We'll have more test mixins eventually, so we are going to end up with a mixed style regardless.